src/main/java/org/pac4j/dropwizard/Pac4JMixins.java

```diff
@@ -0,0 +1,63 @@
+package org.pac4j.dropwizard;
+
+import java.util.List;
+
+import org.pac4j.core.authorization.generator.AuthorizationGenerator;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+public class Pac4JMixins {
+
+    @JsonTypeInfo(use = JsonTypeInfo.Id.CLASS, include = JsonTypeInfo.As.PROPERTY, property = "class")
+    public abstract static class BaseClientMixin {
+
+        @JsonProperty
+        public abstract void setAuthorizationGenerators(List<AuthorizationGenerator> authorizationGenerators);
+
+        @JsonIgnore
+        public abstract void setAuthorizationGenerators(AuthorizationGenerator... authorizationGenerators);
+
+        @JsonIgnore
+        public abstract void setAuthorizationGenerator(AuthorizationGenerator authorizationGenerator);
```

Comment:
I'm not sure this method is necessary, given the varargs alternative above. The one-character difference in the method names could cause confusion.